README.md

```diff
@@ -55,7 +55,8 @@ Here we introduce the National Insurance contribution
            National Insurance contributions: £10.00
 
 Calculation rules:
- * National Insurance contributions: Any amount of money earned above a gross annual salary of £8,060.00 is subject to a National Insurance contribution of 12%
+ * the annual National Insurance (basic contributions): 12% of any amount of money earned above a gross annual salary of £8,060.00.
+ * "National Insurance contributions" on a salary slip is equal to the annual National Insurance (basic contributions) divided by 12.
```

Comment:
There is a mention of `The monthly salary slip should contain the below:` at the top of each of the list of expected fields in the description, if this is sufficient than don't need to mention monthly.